tools/flang2/flang2exe/expsmp.cpp

```diff
@@ -3012,7 +3012,15 @@ exp_mp_func_prologue(void)
     }
   }
 
-  ll_save_gtid_val(bih);
+  if (has_multiple_entries(GBL_CURRFUNC)) {
+    if (bih)
+      ll_save_gtid_val(bih);
+    for (func = gbl.entries; func != NOSYM; func = SYMLKG(func)) {
+      ll_save_gtid_val(BIH_NEXT(BIHNUMG(func)));
```

Comment:
Please reuse `findEnlabBih` here instead of writing `BIH_NEXT(BIHNUMG(func))`.

Reply:
I think you need to update `expb.curbih` here as well, similar to line 3004?

Reply:
I agree, this patch needs refresh. Even the test case is in a wrong directory in which it is never executed.

Reply:
Thanks.